.storybook/preview.tsx

```diff
@@ -66,5 +66,17 @@ export const parameters = {
     ...createPaperReleaseConfig('1.2'),
     ...createPaperReleaseConfig('1.1'),
     ...createPaperReleaseConfig('1.0'),
+    implementationExample: {
+      styles: {
+        backgroundColor: '#ffffff',
+        borderColor: '#000000',
+        color: '#000000',
+      },
+      title: 'Implementation Example',
+      tooltip: {
+        title: `About Implementation Examples`,
+        desc: 'Implementation examples show how you might combine existing EDS components to create custom recipes.',
```

Comment:
should we avoid using the word "recipes" here? maybe "custom layouts" or end the sentence after "EDS components"?

Reply:
agreed, good catch. we might s/combine/compose and chop the end of that sentence off